ios/NeuroidReactnativeSdk.swift

```diff
@@ -60,7 +60,7 @@ class NeuroidReactnativeSdk: NSObject {
 
     @objc(setScreenName:withResolver:withRejecter:)
     func setScreenName(screenName: String, resolve:RCTPromiseResolveBlock,reject:RCTPromiseRejectBlock) -> Void {
-        try? NeuroID.setScreenName(screen: screenName)
+        try? NeuroID.setScreenName(screenName)
```

Comment:
I think we can refactor this to the async/await Bool value instead of a try

Reply:
cleaner to leave this be I think.

Reply:
fixed. Thanks!